### ISSUE TITLE
Build sometimes fails to run rake task

### ISSUE THREAD

**Issue by rdougan** [noseglid/atom-build#82]
I have a simple script setup to run a rake task:

    #!/bin/sh
    rake package mode=intg org=bp target=server

I'm unsure why, but sometimes this works, and sometimes it does not. It appears nothing is changing between these events (often the same day, but Atom has been relaunched).

The error message:

    Executing with sh: ./build-intg.sh 
    (in /Users/Robert/Development/odesk/bestparking/mobileweb)
    rake aborted!
    cannot load such file -- rspec/core/rake_task
    /Users/Robert/Development/odesk/bestparking/mobileweb/rakefile:2:in `<top (required)>'
    (See full trace by running task with --trace)

I assume it doesn't have the right ENV or something, but I haven't been able to figure it out.

Any ideas why this would sometimes work and sometimes does not?

For reference; it always works if I directly run it from Terminal.

Thanks!

**Comment by noseglid** [comment]
Weird. It seems as `rake` suddenly fails. Would it be possible for you to run it with `--trace` as it suggests, to hopefully get some more information?

If the error occurs, do all subsequent builds fails until you reload Atom?

I've done some testing with a shell script executing make, but it works as expected. Even after several reloads.

**Comment by rdougan** [comment]
Thanks!

I think I have figured out the main cause...

I normally open my app folder from **within** the directory. I used to do this using `atom` but it stopped working recently. So I switched to `atom .`

It does not work using `atom .` However I went back to the parent and ran `atom app-folder` and it built! The weird thing is though, if I close atom and reopen the folder from within (`atom .`) - it **still* works....
